build_vol_cmake.sh

```diff
@@ -202,4 +203,28 @@ if [ "${CMAKE_GENERATOR}" = "Unix Makefiles" ]; then
   make -j${NPROCS} && make install || exit 1
 fi
 
+echo "REST VOL (and HDF5) built"
+
+# Clean out the old CMake cache
+rm -f "${BUILD_DIR}/CMakeCache.txt"
+
+# Configure vol-tests
+
+mkdir -p "${BUILD_DIR}/tests/vol-tests"
+cd "${BUILD_DIR}/tests/vol-tests"
+
+if [ -z "$PREBUILT_HDF5_DIR" ]; then
+    HDF5_INSTALL_DIR=$HDF5_DIR
+else
+    HDF5_INSTALL_DIR=$PREBUILT_HDF5_DIR
+fi
+
+CFLAGS="-D_POSIX_C_SOURCE=200809L" cmake -G "${CMAKE_GENERATOR}" -DHDF5_DIR=${HDF5_INSTALL_DIR} -DCMAKE_INSTALL_PREFIX="${INSTALL_DIR}" "${CONNECTOR_DEBUG_OPT}" "${CURL_DEBUG_OPT}" "${MEM_TRACK_OPT}" "${THREAD_SAFE_OPT}" "${SCRIPT_DIR}/test/vol-tests"
```

Comment:
We should probably revisit this when we switch to the API tests. I'm not sure that this flag should really be needed.